docker-compose.yaml

```diff
@@ -35,6 +37,26 @@ services:
     links:
       - "db:database"
 
+  chatroom_websocket:
+    build:
+      context: .
+      dockerfile: dockerfile.chat_app_socket_server
+    container_name: chatroom_websocket
+    depends_on:
+      - db
+    environment:
+      HOST: 0.0.0.0
+      PORT: 8085
+      # MYSQL_DB_HOSTS: mysql:3306
+      # MYSQL_DB_NAME: userdb
+      # MYSQL_DB_USERNAME: root
```

Comment:
This Dockerfile change is also acceptable, but there is another way to build binary files during the 'make build' process and run binary files with the help of bash scripts. The consumption has the same implementation

Reply:
Will look into this.